examples/lammps/cr_rlc_lammps_pkg/_trajectory_callback.py

```
@@ -0,0 +1,119 @@
+                var_name = var_names[j]
+                if var_name in ctrl_map:
+                    lb, ub = ctrl_map[var_name]
+                    if lb and ub:
+                        for _nstep in range(args["nstep"], args["nstep"] + ncountmax):
+                            val = lmp.extract_fix(fix_id, 0, 2, _nstep, j)
+                            if val <= lb or val >= ub:
+                                raise cr.CRError(
+                                    f'the "{var_name}"\'s value = {val} is '
+                                    f"out of bound of ({lb} {ub}). This run "
+                                    "is unstable."
+                                )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)